cannot find file \Revive\Revive\ReviveInjector_x64.exe

Running the desktop shortcut, I see a momentary loading wheel (less than a second) and then nothing.
Normal injection works fine. I've confirmed the paths are correct and confirmed clicking "Save and Exit".

In the error.log is this reference to ReviveInjector_x64.exe:
```
Exception in thread "main" java.io.IOException: Cannot run program "D:\Program Files\Revive\Revive\ReviveInjector_x64.exe" (in directory "D:\Program Files\Revive\Revive"): CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	at com.cabbagedevelopment.condor2helper.CondorLauncher.launch(CondorLauncher.kt:38)
	at com.cabbagedevelopment.condor2helper.MainApplicationKt.main(MainApplication.kt:30)
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessImpl.create(Native Method)
	at java.lang.ProcessImpl.<init>(ProcessImpl.java:386)
	at java.lang.ProcessImpl.start(ProcessImpl.java:137)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 2 more
```

Poking around the Revive install folder I don't see a ReviveInjector_x64.exe file anywhere. The only contents of D:\Program Files\Revive\Revive\ are the x64 and x86 sub dirs. (Confirmed this is Revive v1.9.1.)

I tried moving the contents of ...\Revive\Revive\x64 to ...\Revive\Revive and renaming ReviveInjector.exe to ReviveInjector_x64.exe. That results in an empty error.log and suspended Condor2 object in Windows Task Manager. (screenshot 
https://drive.google.com/open?id=1rkT91WUYY2zk33NZl_upD5wvl_4QYYlx)

--- a/src/main/kotlin/com/cabbagedevelopment/condor2helper/MainApplication.kt
+++ b/src/main/kotlin/com/cabbagedevelopment/condor2helper/MainApplication.kt
@@ -39,7 +39,7 @@ fun main(args: Array<String>) {
  */
 class MainApplication : App(MainView::class) {
 
-    val injectorName = "ReviveInjector_x64.exe"
+    val injectorName = "ReviveInjector.exe"
     val prefs = Prefs()
     val view = find<MainView>()
 
@@ -86,7 +86,6 @@ class MainApplication : App(MainView::class) {
     fun save() {
         prefs.condorPath = view.condorText.text
         prefs.revivePath = view.reviveText.text
-        prefs.reviveInjectorPath = "${prefs.revivePath}\\$injectorName"
+        prefs.reviveInjectorPath = "${prefs.revivePath}\\x64\\$injectorName"
     }
 }
-

